pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -320,6 +339,24 @@ func activateTenant(ctx context.Context, execCtx interface{}, newTenantID roachp
 	})
 }
 
+func (s *streamIngestionResumer) cancelProducerJob(
```

Comment:
We don't have to do it in this PR. But I wonder if this should be configurable. Since this will release the protected timestamp on the producer side.